SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1623,7 +1623,7 @@ void USpatialNetDriver::TickFlush(float DeltaTime)
 	double ServerReplicateActorsTimeMs = 0.0f;
 #endif // USE_SERVER_PERF_COUNTERS
 
-	if (IsServer() && GetSpatialOSNetConnection() != nullptr && PackageMap->IsEntityPoolReady())
+	if (IsServer() && GetSpatialOSNetConnection() != nullptr && PackageMap->IsEntityPoolReady() && bIsReadyToStart)
```

Comment:
This is a bit of a scary change, but it worked in all my testing. I was seeing CreateEntity sent to Spatial via the tick processing here, even before the Translator/NetDriver had finished startup processing. Then it would create entities with invalid authority intents.

Reply:
LGTM but might be worth pulling in someone more experienced with GDK to eyeball?

Reply:
Looks fine. I think we can remove the entitypool is ready check also. cc @MatthewSandfordImprobable